From 580af869093cfda6bc8a9d5901850354a16b3666 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tim=20R=C3=BChsen?= <tim.ruehsen@gmx.de>
Date: Fri, 25 Dec 2020 18:55:34 +0100
Subject: [PATCH] * configure.ac: Use AC_CONFIG_MACRO_DIR only once

This fixes #547 (autotools 2.70 is more restrictive).
Reported by: Gabriele Balducci
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 7e29b8cba..0a4dc05b9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -100,7 +100,6 @@ AC_CONFIG_FILES([include/wget/wgetver.h])
 
 AC_CONFIG_SRCDIR([src/wget.c])
 AC_CONFIG_HEADERS([config.h])
-AC_CONFIG_MACRO_DIR([m4])
 
 # Non-verbose make
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
-- 
GitLab

